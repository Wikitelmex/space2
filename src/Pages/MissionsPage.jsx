import React from 'react';

const MissionsPage = () => {
    const x = "missions page under construction";
    return (
        <div>
            <p>{x}</p>
        </div>
    );
}

export default MissionsPage;