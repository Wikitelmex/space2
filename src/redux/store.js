import { createStore, applyMiddleware, combineReducers } from 'redux';
import logger from 'redux-logger';
import thunk from 'redux-thunk';

// import reducers here
import missionsReducer from './missions/missionReducer.js';

const mixReducer = combineReducers({
  // add reducers here
  missions: missionsReducer,
});

const store = createStore(
  mixReducer,
  applyMiddleware(logger, thunk),
);

export default store;
