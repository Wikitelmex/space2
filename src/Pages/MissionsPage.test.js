import React from "react";
import { render, cleanup } from "@testing-library/react";
import MissionsPage from "../Pages/MissionsPage";

import { Provider } from 'react-redux';
import store from '../redux/store';

afterEach(cleanup);

it("renders mission page with redux", () => {
  const { asFragment } = render(
    <Provider store={store}>
      <MissionsPage />
    </Provider>);
  expect(asFragment()).toMatchSnapshot();
});