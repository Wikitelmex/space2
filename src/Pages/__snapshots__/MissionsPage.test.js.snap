// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders mission page with redux 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;

exports[`renders with redux 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;
